_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -21,14 +21,22 @@ Electron-based wallet called “Daedalus”.
 The source code for both Cardano SL and Daedalus Bridge can be obtained
 from the [official repository](https://github.com/input-output-hk/cardano-sl).
 
-The [Haskell Tool Stack](https://haskellstack.org) is required to build Cardano SL. Furthermore, we strongly suggest using [Nix package manager](https://nixos.org/nix/download.html) to get the correct dependencies for building Cardano SL. It will fetch the correct `openssl` version, but won't override the system-installed version. The following commands assume that you already has `stack` and `nix-*` programs.
+Cardano SL supports three ways for building itself:
```

Comment:
We also have a docker image and should be noted as supported way.

Reply:
@domenkozar, should we get Michael to have a pass, so he documents it?